packages/@aws-cdk/aws-cloudformation/test/integ.core-cross-region-references.ts

```diff
@@ -0,0 +1,65 @@
+import { Queue, IQueue } from '@aws-cdk/aws-sqs';
```

Comment:
Put the integration test here since we can't have integration tests in `core` and there is a precedent for doing it this way.